flang/lib/Lower/ConvertVariable.cpp

```diff
@@ -124,6 +125,23 @@ static bool isConstant(const Fortran::semantics::Symbol &sym) {
          sym.test(Fortran::semantics::Symbol::Flag::ReadOnly);
 }
 
+/// Is this a compiler generated symbol to describe derived types ?
+static bool isRuntimeTypeInfoData(const Fortran::semantics::Symbol &sym) {
+  // So far, use flags to detect if this symbols were generated during
```

Comment:
I think you meant to say "if this symbol were" rather than if this symbols were".